flink-runtime/src/main/java/org/apache/flink/runtime/webmonitor/WebMonitorEndpoint.java

```diff
@@ -179,6 +189,7 @@
     private final LeaderElectionService leaderElectionService;
 
     private final FatalErrorHandler fatalErrorHandler;
+    private final ThreadInfoOperatorTracker<OperatorThreadInfoStats> threadInfoOperatorTracker;
```

Comment:
I think we could create this within initializeHandlers(); there isn't really a need to have this here.

Reply:
moved as requested.